src/sc_kpm/sc_agent.py

```diff
@@ -49,6 +49,9 @@ def unregister(self) -> None:
             self._logger.warning("Event's already destroyed or not registered")
         self._logger.info("Agent's unregistered")
 
+    def callback(self, class_node: ScAddr, edge: ScAddr, action_node: ScAddr) -> ScResult:
+        return self.on_event(class_node, edge, action_node)
```

Comment:
Can we make callback private? I think user shouldn't be able to use it

Reply:
Done